extra-electronics/bsc/autobuild/build

```diff
@@ -0,0 +1,21 @@
+abinfo "Install dependencies ..."
+cabal v1-update
+cabal v1-install regex-compat syb old-time split
+
+abinfo "Building ..."
+make -C "$SRCDIR" release NOASCIIDOCTOR=1
```

Comment:
Was `-C "$SRCDIR"` necessary?

Reply:
No, but I think adding "$SRCDIR" everywhere is ALL not necessary.

We shouldn't add "$SRCDIR" at all when $PWD = $SRCDIR, I think.

If you're doubting this, please rethink every $SRCDIR usage.